heartbeat/podman-etcd

```diff
@@ -453,6 +521,64 @@ prepare_env() {
 	LISTEN_METRICS_URLS="0.0.0.0"
 }
 
+
+generate_etcd_configuration() {
+	if is_force_new_cluster; then
+		# The embedded newline is required for correct YAML formatting.
+		FORCE_NEW_CLUSTER_CONFIG="force-new-cluster: true
+force-new-cluster-bump-amount: 1000000000"
```

Comment:
what is this for?

Reply:
This is something we left out initially. It's part of the restore-quorum script in CEO. The script is more advanced, calculating the bump automatically. I had a simpler change for testing, which I forgot to complete.

https://github.com/openshift/cluster-etcd-operator/commit/7a5c8910796e6314d0af6ff715eeb1ef522b25ba